.github/workflows/test.yml

```diff
@@ -13,12 +13,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
 
       - name: Install Node.js
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
-          node-version: 16
+          node-version: 18
```

Comment:
Why Node 18 here and Node 20 below?

Reply:
This node version is whats executing the package.json scripts where we are using node 18.7.6.  The node20 is used by github actions runner which there is no node18. It goes from node16 to node20 and node16 is being deprecated in actions.